returnn/datasets/basic.py

```diff
@@ -874,37 +874,6 @@ def is_less_than_num_seqs(self, n):
     # in advance can handle this somehow.
     return n < self.num_seqs
 
-  def can_serialize_data(self, key):
-    """
-    :param str key: e.g. "classes"
-    :rtype: bool
-    """
-    return key in self.labels
-
-  def serialize_data(self, key, data):
-    """
-    :param str key: e.g. "classes". self.labels[key] should be set
-    :param numpy.ndarray data: 0D or 1D
-    :rtype: str
-    """
-    labels = self.labels[key]
-    if data.ndim == 0:
-      data = numpy.expand_dims(data, axis=0)
-    assert data.ndim == 1
-    if len(labels) < 1000 and all([len(label) == 1 for label in labels]):
-      # are these actually raw bytes? -> assume utf8
-      if all([ord(label) <= 255 for label in labels]):
-        try:
-          if PY3:
-            return bytes([ord(labels[c]) for c in data]).decode("utf8")
-          else:
-            return b"".join([bytes(labels[c]) for c in data]).decode("utf8")
-        except UnicodeDecodeError:
-          pass  # pass on to default case
-      return "".join(map(labels.__getitem__, data))
-    else:
-      return " ".join(map(labels.__getitem__, data))
-
```

Comment:
I think I would not remove these functions, in case some people have some scripts which rely on these.
Rather, `serialize_data` can simply call to `get_seq_labels_from_labels`.